logging/performance_reliability/logging-flow-control-mechanisms.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-flow-control-mechanisms"]
+= Flow control mechanisms
+:context: logging-flow-control-mechanisms
+
+toc::[]
+
+If logs are being produced faster than they can be collected, it can be difficult to predict or control the volume of logs to an output.
+
+This can result in logs being lost, or in the event of a system outage, when log buffers build up without user control, this can cause long recovery times and high latency when the connection is restored.
+
+As an administrator, you can configure flow control mechanisms for your {logging}. You can choose to limit logging rates, or ignore some logs entirely.
+Logs may still be lost if the collector cannot keep up, but when using flow control mechanisms, you have greater control over which logs are lost, and greater predictability of log rates.
```

Comment:
I think I would drop this line entirely.  Cases do exist where the collector can not keep up but I have concerns we will be asked to expand upon these use cases.  One of the primary benefits of this feature is "fair share".  It's the case where a verbose service overwhelms collection to the point that logs from more important services may not be collected in a timely manor.

Reply:
I think fair share is pretty much covered in the "benefits" section below - removing this statement